getting-started/web-app.md

```diff
@@ -20,10 +20,30 @@ This article explains how to use the Telerik UI for Blazor components into a <a
 
 1. Select the [**Blazor Web App**](https://learn.microsoft.com/en-us/aspnet/core/blazor/project-structure?view=aspnetcore-8.0#blazor-web-app) project type, enter a name for your project, and then click **Next**.
 
-1. Select the .NET 8 framework, the desired [interactive render mode](https://learn.microsoft.com/en-us/aspnet/core/blazor/components/render-modes?view=aspnetcore-8.0#render-modes) and interactivity location (per page or global).
+1. Select the desired [Interactive render mode](https://learn.microsoft.com/en-us/aspnet/core/blazor/components/render-modes?view=aspnetcore-8.0#render-modes) and Interactivity location (*Global* or *Per page / component*). See the [notes below](#interactivity-considerations) while choosing Interactivity location.
```

Comment:
* Consider using only the "happy path", which seems to be **Global**. You can reference to another article that describes the **Per page/component** approach. 
* As a general rule, use **bold** when referring to menus, labels, dropdowns, etc. that appear in the UI.
* Keep the style unchanged when describing options in the UI (Per page / component vs. Per page/component).
```suggestion
1. Select the desired [Interactive render mode](https://learn.microsoft.com/en-us/aspnet/core/blazor/components/render-modes?view=aspnetcore-8.0#render-modes) and Interactivity location (**Global** or **Per page/component**). See the [notes below](#interactivity-considerations) while choosing Interactivity location.
```